src/helpers.js

```diff
@@ -101,6 +102,8 @@ export async function queryWebsoc(params) {
         }).then((res) => res.json());
         websocCache[searchString] = backupResponse;
         return backupResponse;
+    } finally {
+        websocCache[searchString].timestamp = Date.now();
```

Comment:
Let's avoid using `finally` here by doing something like this:
```suggestion
websocCache[searchString] = { ...backupResponse, timestamp: Date.now() };
return backupReponse;
```

Reply:
I used `finally` there because it should be doing the same thing (adding timestamp) regardless of whether the try or catch runs, and doing it without `finally` would make for duplicate code (same thing for lines 95 and 103). What is the reason we should be avoiding `finally`?

Reply:
Ah I didn't see what was in the try block, since I was on my phone. Makes sense not to duplicate code by using finally 👍